src/utils/appEngagementMonitor/appEngagementMonitor.js

```diff
@@ -0,0 +1,30 @@
+const { query: Hasura } = require('../hasura');
+
+const userInactivityTracker = async () => {
+  try {
+    // TODO: Fetch user_actions data from the DB
+
+    for (const user of users.rows) {
+      const { user_id, last_app_opened_timestamp } = user;
+
+      const today = new Date();
+      const appOpenedToday = last_app_opened_timestamp >= today.setHours(0, 0, 0, 0) && last_app_opened_timestamp <= today.setHours(23, 59, 59, 999);
```

Comment:
Can the timestamp ever be greater than end of today (23, 59, 59, 999)?

Reply:
It wont be right?

Reply:
and if it would be it would count as next day